mods/ra/rules/civilian.yaml

```diff
@@ -244,8 +244,9 @@ V19.Husk:
 
 BARL:
 	Inherits: ^TechBuilding
-	Selectable:
-		Priority: 0
+	-Selectable:
+	CustomSelectionSize:
+		CustomBounds: 24,24
```

Comment:
This is a bit large for a single barrel. On the other hand it makes target clicking them easier.